src/components/formik-checkbox/FormikCheckbox.tsx

```diff
@@ -113,7 +106,6 @@ export class FormikCheckbox extends React.Component<Props> {
           </Checkbox>
           <Label dangerouslySetInnerHTML={{__html: label}} error={!!errors}></Label>
         </CheckBoxWrapperStyled>
-        {errors && <ErrorMessageStyled>{errors}</ErrorMessageStyled>}
       </WrapperStyled>
```

Comment:
Waarom error message weg halen bij Checkbox? Als je ze vergeet aan te klikken dan moet je toch error state krijgen?

Reply:
Hebben we vorige week hier over gehad, en besloten dat het voldoende is dat het veldje gewoon de error kleur krijgt, zonder begeleidende errorboodschap. (ziet er een beetje overkill uit anders)

Reply:
Oh oké dus nergens nog begeleidende boodschap of ?